devtools/archive/jenkins/results.table

```diff
@@ -1,386 +0,0 @@
-Index splitting
```

Comment:
Would you be willing to make an issue in the https://github.com/deepchem/moleculenet and paste in this table for reference? @mufeili has been taking the lead on getting some new benchmark numbers up and these old numbers may be a useful sanity check for him as he's benchmarking

Reply:
OK 👍  I made deepchem/moleculenet#18.